drivers/growatt/device.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019: Rob, https://github.com/pro-sumer
+// License: GNU GPLv3, https://www.gnu.org/licenses/gpl.txt
+
+'use strict'
+
+const Inverter = require('../inverter')
+const api = require('./api')
+
+class Growatt extends Inverter {
+  async checkProduction () {
+    try {
+      const settings = this.getSettings()
+      await api.login(settings.username, settings.password)
+      const productionResponse = await api.getProduction()
+      const productionData = await productionResponse.json()
+      if (productionData.plantNumber === 1) {
+        if (!this.getAvailable()) {
+          this.setAvailable().then(result => {
+            this.log('Available')
```

Comment:
For consistency, it would be nice to use async/await syntax here as well, instead of promises.

Reply:
Changed to use `await` (code copied from other driver in SolarPanels)